rasa/shared/nlu/training_data/training_data.py

```diff
@@ -432,7 +412,7 @@ def get_nlg_persist_filename(nlu_filename: Text) -> Text:
             # we are going to dump in the same format as the NLU data. unfortunately
             # there is a special case: NLU is in json format, in this case we use
             # md as we do not have a NLG json format
```

Comment:
This comment referencing `md` should be updated too.

Reply:
Ah, this didn't get updated:
```suggestion
            # yaml as we do not have a NLG json format
```